elastic/kibana
# [Event Log] Add `consumer`, `rule_type_id` and `space_id` to event log documents

Resolves https://github.com/elastic/kibana/issues/127623

## Summary

Added new fields to support RBAC on event log documents.

* Adds rule consumer to `kibana.alert.rule.consumer` event log field
* Adds rule type id to `kibana.alert.rule.rule_type_id` event log field
* Adds space id to `kibana.space_ids` event log field

## To Verify

* Create a rule that generates alerts and schedules actions for execution. Let it run and verify that the event log documents that are written out contain the new fields for `consumer`, `rule_type_id` and `space_ids`. These fields should be in all event log documents, including the docs for action execution.

### Checklist
